5] dice/main.py

```diff
@@ -0,0 +1,39 @@
+from random import randint
+
+dice = ("D3", "D4", "D6", "D8", "D10", "D12", "D20", "D100")
+
+
+def roll_the_dice(input_dice):
+    for cube in dice:
+        if cube in input_dice:
+            try:
+                number_of_rolls, modifier = input_dice.split(cube)
+            except ValueError:
+                return "Wrong Dice"
+            type_of_dice = cube[1:]
+            break
+    else:
+        return "Wrong dice"
+
+    if number_of_rolls == "":
```

Comment:
doporuil bych spis `if not number_of_rolls:` -> prazdny string ma bool hodnotu False